crates/astria-sequencer/src/app/storage/values/revision_number.rs

```diff
@@ -0,0 +1,51 @@
+use std::fmt::{
+    self,
+    Display,
+    Formatter,
+};
+
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+
+use super::Value;
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(crate) struct RevisionNumber(u64);
```

Comment:
We should have a domain type for this if it requires an on-disk type.

Reply:
Ticketed at #1599.